maptk/geo_reference_points_io.cxx

```diff
@@ -99,20 +99,20 @@ void load_reference_file(vital::path_t const& reference_file,
     // input landmarks are given in lon/lat/alt format (ignoring alt for now)
     ss >> vec;
 
-    // When this is called the first time, setzone is given a -1, which is the
-    // default for the function.
-    lgcs.geo_map_algo()->latlon_to_utm(vec.y(), vec.x(), x, y, zone, northp,
-                                       lgcs.utm_origin_zone());
-    vec[0] = x; vec[1] = y; vec[2] = vec.z();
-    mean += vec;
-
-    // Use the zone of the first input landmark as the base zone from which we
-    // interpret all other geo-positions with respect to.
-    if (set_lgcs_origin)
+    vital::vector_2d lon_lat(vec.x(), vec.y());
```

Comment:
Maybe it is just because I am unfamiliar with the code, but I see that `write_local_geo_cs_to_file` appears to be writing lat, lon, which is what triggered my suspicion... Are these dealing with different files? Is it intended that these two spots are different?